lib/matplotlib/backends/backend_pdf.py

```diff
@@ -424,7 +435,25 @@ class PdfFile:
     """PDF file object."""
 
     def __init__(self, filename, metadata=None):
-        self.nextObject = 1     # next free object id
+        """
+        Parameters
+        ----------
+
+        filename : file-like object or string
+            output target; if a string, a file will be opened for writing
+        metadata : dict from strings to strings and dates
+            Information dictionary object (see PDF reference section 10.2.1
+            'Document Information Dictionary'), e.g.:
+            `{'Creator': 'My software', 'Author': 'Me',
+            'Title': 'Awesome fig'}`
+
+            The standard keys are `'Title'`, `'Author'`, `'Subject'`,
+            `'Keywords'`, `'Creator'`, `'Producer'`, `'CreationDate'`,
+            `'ModDate'`, and `'Trapped'`. Values have been predefined
+            for `'Creator'`, `'Producer'` and `'CreationDate'`. They
+            can be removed by setting them to `None`.
+        """
+        self.objectId = itertools.count(1)  # consumed by reserveObject
```

Comment:
Maybe better name this
```suggestion
        self._object_id_generator = itertools.count(1)  # consumed by reserveObject
```
?

Reply:
I gave the new names `_seq` suffixes to keep the lines slightly shorter.